integration/spark/src/main/common/java/io/openlineage/spark/agent/facets/TableProviderFacet.java

```diff
@@ -0,0 +1,24 @@
+package io.openlineage.spark.agent.facets;
+
+import static io.openlineage.spark.agent.client.OpenLineageClient.OPEN_LINEAGE_CLIENT_URI;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.openlineage.client.OpenLineage;
+import lombok.Getter;
+import lombok.NonNull;
+
+@Getter
+public class TableProviderFacet extends OpenLineage.DefaultDatasetFacet {
```

Comment:
I'm unclear what this facet is supposed to communicate. Can you add some docs?

Reply:
Sure.